src/index.html

```diff
@@ -1,13 +1,49 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="card-block" data-qa="card">
+      <img
+        src="images/imac.jpeg"
+        alt="i-mac"
+        class="mac-image"
+      >
+      <h2 class="advert-title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+      <div class="product-code">
+        Product code: 195434
+      </div>
+      <div class="product-review">
+        <div class="stars-container">
+          <div class="star"></div>
+          <div class="star"></div>
+          <div class="star"></div>
+          <div class="star"></div>
+          <div class="star"></div>
+        </div>
+        <div class="reviews">Reviews: 5</div>
+      </div>
+      <div class="price-container">
+        <div class="price-label">Price:</div>
+        <div class="price">$2,199</div>
+      </div>
+      <a
+        href="#"
```

Comment:
You should use `href="#buy"` or something like that, because, according to the checklist:

> Item with the 'Buy' text should be a link.